### ISSUE TITLE
Publish npm package

### ISSUE THREAD

**Issue by junderw** [bitcoinjs/bip44-constants#32]
Just realized I am not a publisher for this package:

https://www.npmjs.com/package/bip44-constants

Tasks:

1. Pull current github master
2. Run npm publish

I already tagged the commit and pushed the tag for v8.0.0 all we need is the publish.

**Comment by junderw** [comment]
Here's the info I got before it told me I have no permissions
```
npm notice 📦  bip44-constants@8.0.0
npm notice === Tarball Contents === 
npm notice 787B   package.json
npm notice 1.7kB  CHANGELOG.md
npm notice 16.3kB index.js    
npm notice 1.1kB  LICENSE     
npm notice 1.7kB  README.md   
npm notice === Tarball Details === 
npm notice name:          bip44-constants                         
npm notice version:       8.0.0                                   
npm notice package size:  7.8 kB                                  
npm notice unpacked size: 21.6 kB                                 
npm notice shasum:        fefc814d32b5e555861725dba34f5c1b9bab55e7
npm notice integrity:     sha512-zeHgj5DGlZnu/[...]v6ONMdr5Qlwfg==
npm notice total files:   5
```

**Comment by jprichardson** [comment]
@junderw added you.

**Comment by junderw** [comment]
Published to v8.0.0
